packages/babel-plugin-flow-runtime/src/createConversionContext.js

```diff
@@ -33,5 +35,14 @@ export default function createConversionContext (options: Options): ConversionCo
                          ? true
                          : Boolean(options.annotate)
                          ;
+  
+  if ('suppressComments' in options && Array.isArray(options.suppressComments)) {
+    context.suppressCommentPatterns = options.suppressComments.map(RegExp);
```

Comment:
This won't work - `map` will pass the array index as the second argument to `RegExp`, and this will throw - the second argument indicates the RegExp flags.